tailwind.config.js

```diff
@@ -0,0 +1,82 @@
+const glob = require("glob");
+const path = require("path");
+
+const decanter = path.resolve(__dirname, "node_modules/decanter/src/plugins");
+
+// Import 3rd party tailwind plugins.
+let plugins = [
+  require("@tailwindcss/aspect-ratio"),
+  require('@tailwindcss/forms'),
+  require("tailwindcss-children"),
+  require("tailwindcss-interaction-variants"),
+];
+// Load theme customizations.
+const theme = {};
+const themeFiles = glob.sync(decanter + "/theme/*.js");
+themeFiles.forEach((file) => {
+  const matches = file.match(/\/([^\/]+).js/);
+  const propName = matches[1];
+  theme[propName] = require(file);
+});
+
+// Prepare safelist for PurgeCSS
+const safelist = [];
+
+Object.entries(theme.colors()).forEach(([property, value]) => {
+  if (typeof value === "string") {
+    safelist.push(`su-bg-${property}`);
+    safelist.push(`su-text-${property}`);
+  } else if (typeof value === "object") {
+    Object.entries(value).forEach(([childProp, childValue]) => {
+      if (childProp === "DEFAULT") {
+        safelist.push(`su-bg-${property}`);
+        safelist.push(`su-text-${property}`);
+      } else {
+        safelist.push(`su-bg-${property}-${childProp}`);
+        safelist.push(`su-text-${property}-${childProp}`);
+      }
+    });
+  }
+});
+
+// Load all base plugins from Decanter.
+const base = [];
+const baseFiles = glob.sync(decanter + "/base/*.js");
+baseFiles.forEach((file) => {
+  base.push(require(file)());
+});
+plugins = plugins.concat(base);
+
+// Load all component plugins from Decanter.
+const componentFiles = glob.sync(decanter + "/components/**/*.js");
+const components = [];
+componentFiles.forEach((file) => {
+  if (file.match(/logo/)) {
+    return;
+  }
```

Comment:
@nplowman This all looks good to me. Just a question - why is the logo file treated differently?

Reply:
Thanks for catching this. I didn't intend to commit this - I was running into errors when it hit the logo plugin, and added this in while I was debugging. I'll remove this.

Reply:
Awesome - thank you, @nplowman !

Reply:
@yvonnetangsu Debug code has been removed.